monai/engines/evaluator.py

```diff
@@ -53,6 +53,8 @@ class Evaluator(Workflow):
             engine.state.metrics when epoch completed. key_val_metric is the main metric to compare and save the
             checkpoint into files.
         additional_metrics: more Ignite metrics that also attach to Ignite Engine.
+        metric_cmp_fn: function to compare current key metric with previous best key metric value.
+            it must accept 2 args (current_metric, previous_best) and return a bool result. default to `greater than`.
```

Comment:
```suggestion
            It must accept 2 args (current_metric, previous_best) and return a bool result. Default to `greater than`.
```
Same for others below. 

 I think the docstring is missing the explanation on what to return and how it will taken into account by the code. For example, if returns True, what will happen.

Reply:
Hi @vfdev-5 , thanks for your review! I updated according to your comments.